rotkehlchen/chain/ethereum/modules/nft/nfts.py

```diff
@@ -193,16 +193,13 @@ def get_db_nft_balances(self, filter_query: 'NFTFilterQuery') -> Dict[str, Any]:
             'total_usd_value': total_usd_value,
         }
 
-    def get_balances(
+    def fetch_nft_balances(
             self,
             addresses: List[ChecksumEvmAddress],
             uniswap_nfts: Optional[AddressToUniswapV3LPBalances],
-            return_zero_values: bool,
-            ignore_cache: bool,
-    ) -> Dict[ChecksumEvmAddress, List[Dict[str, Any]]]:
+    ) -> None:
         """Gets all NFT balances. The actual opensea querying part is protected by a lock.
```

Comment:
What I would ask is to edit the docstring to specify that this function queries nft balances but does not return them. Since both get, and fetch mean the same thing.